Lesson7/organizer7/scheduler/templates/admin_user_list_base.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>User List</title>
+</head>
+
+<body>
+    {% block main_part %}
+        {%  if users %}
+            <ul>
+                {% for user in users %}
+                    <li>
+                        User:
+                        <a href=" {{ user.name }}/">
```

Comment:
It would be better to use Django template tag `url` https://docs.djangoproject.com/en/4.2/ref/templates/builtins/#url